.travis.yml

```diff
@@ -125,9 +124,20 @@ jobs:
 
       env: APP_VERSION=${TRAVIS_TAG#v}
 
+      before_install:
+        - ./scripts/pull_travis_ci_tools.sh
+      install:
+        # installs hub to ~/bin
+        - $HOME/travisci-tools/release_github/install_hub.sh
+        - export PATH=$HOME/bin:$HOME/travisci-tools/release_github:$PATH
+
       script:
         # now we're going to create packages & upload packages
         - $TRAVIS_BUILD_DIR/scripts/ci_create_packages.sh && $TRAVIS_BUILD_DIR/scripts/ci_upload_packages.sh
+        # create the github release
+        - release_github_v2.sh "$TRAVIS_TAG"
+        # attach generate_secret to the github release
+        - $TRAVIS_BUILD_DIR/scripts/ci_build_generate_secret.sh && hub release edit "$TRAVIS_TAG" -m "" -a "/tmp/output_packages/generate_secret#generate_secret for Linux 64-bit"
```

Comment:
Should we build for OSX and Windows too?

Reply:
yea ill work on that next